api/integrations/codebase/zulip_codebase_mirror

```diff
@@ -74,6 +74,7 @@ while len(json_implementations):
         continue
 
 def make_api_call(path):
+    # type: (unicode) -> Optional[List[Union[unicode, Tuple[str]]]]
```

Comment:
Don't you think unicode won't work with python 3, you could change it to `typing.Text`